grails-app/migrations/views/invoice-candidate-view.sql

```diff
@@ -52,5 +52,6 @@ union
           join order_shipment on order_shipment.shipment_item_id = shipment_item.id
           join order_item on order_shipment.order_item_id = order_item.id
           join `order` on order_item.order_id = `order`.id
+          left join order_invoice on order_invoice.order_item_id = order_item.id
           left join shipment_invoice on shipment_invoice.shipment_item_id = shipment_item.id
- where shipment_invoice.invoice_item_id is null and shipment.current_status = 'SHIPPED');
+ where order_invoice.invoice_item_id is null and shipment_invoice.invoice_item_id is null and shipment.current_status = 'SHIPPED');
```

Comment:
Each condition of the where clause should be on a separate line for readability.